pacbio-subreads-processing.wdl

```diff
@@ -21,175 +21,224 @@ version 1.0
 # SOFTWARE.
 
 import "structs.wdl" as structs
-import "tasks/bam2fastx.wdl" as bam2fastx
-import "tasks/ccs.wdl" as ccs
+import "bam2fastx.wdl" as bam2fastx
+import "ccs.wdl" as ccs
 import "tasks/fastqc.wdl" as fastqc
-import "tasks/isoseq3.wdl" as isoseq3
-import "tasks/lima.wdl" as lima
+import "samtools.wdl" as samtools
+import "isoseq3.wdl" as isoseq3
+import "lima.wdl" as lima
 import "tasks/multiqc.wdl" as multiqc
+import "pbbam.wdl" as pbbam
 
 workflow SubreadsProcessing {
     input {
-        File subreadsConfigFile
-        String outputDirectory = "."
```

Comment:
How come you want to remove the `outputDirectory`?

Reply:
Just further simplification. If there is only a single subreads bamfile, there is no need for the output directory to have a specific name.

Reply:
This can also be used for the location of the total output, for instance: `/exports/sasc/project/analysis/pacbio`.
As a alternative to the `cromwell run -o` optionl. It won't cause problems to keep this in right?

Reply:
This can still be useful when I have to run multiple subreads bamfiles like my previous project.
I can define a directory per subreads file in a top workflow that scatters this pipeline.